Modules/III-k6-Intermediate/08-Setting-load-profiles-with-executors/Constant-Arrival-Rate-Exercises.md

```diff
@@ -2,39 +2,41 @@
 
 As noted in [Setting load profiles with executors](../08-Setting-load-profiles-with-executors.md#Constant-Arrival-Rate), the _Constant Arrival Rate_ executor has a primary focus on the _iteration rate_ being applied over a specified timeframe.
 
+This is a scenario typically seen with [load testing an API](https://k6.io/docs/testing-guides/api-load-testing/) when there is a need to simulate a constant request rate for a particular API endpoint.
+
 ## Exercises
 
-For our exercises, we're going to start by using a very basic script that simply performs an HTTP request and then waits one second before completing the test iteration. We're providing some console output as things change.
+For our exercises, we're going to start by using a basic script that simply performs an HTTP request on each test iteration and aims at achieving 50 requests per time unit (which defaults to 1 second). We're providing some console output as things change.
```

Comment:
```suggestion
Our exercises start with a basic script. It runs an HTTP request on each test iteration and tries to achieve 50 requests per time unit (which defaults to 1 second). We provide some console output as things change.
```

Reply:
Shortening sentences. Also, avoid "simple" as an adjective because it's subjective.